verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -165,7 +156,7 @@ const StyledSearchTextField = withStyles({
     },
 })(TextField);
 
-function SearchBar(props: {
+export function SearchBar(props: {
```

Comment:
Now it feels like this would live better in its own file, up to you.

Reply:
Yea agreed. Moved.